### PR TITLE
Fix issue #33464 [processor/groupbytrace] migrate from opencensus library

### LINKED ISSUE #33464: Migrate from opencensus library: processor/groupbytraceprocessor

### PR DESCRIPTION
**Pull request by honeychaudharyc**
This PR migrates the internal metrics from opencensus for the groupbytrace processor to  Opentelemetry for metrics, via mdatagen. Used metadata.Telemetrybuilder and replaced all metrices from metrics file to metadata.yaml file. Updated tests to move from opencensus.

Fixes [#33464](https://github.com/open-telemetry/opentelemetry-collector-contrib/issues/33464)

Testing: Unit tests

Documentation: Documentation generated by metadata.yaml.

### CODE REVIEW

**Review by codeboten — COMMENTED** [review]
Thanks @honeychaudharyc, please sign the CLA and we can get this reviewed

File: processor/groupbytraceprocessor/documentation.md [hunk 1]
```diff
@@ -0,0 +1,71 @@
+[comment]: <> (Code generated by mdatagen. DO NOT EDIT.)
+
+# groupbytrace
+
+## Internal Telemetry
+
+The following telemetry is emitted by this component.
+
+### processor_groupbytrace_conf_num_traces
```
Comment (jpkrohling) [inline processor/groupbytraceprocessor/documentation.md:9]: I think we have an otelcol_ prefix to this metric today. We need to ensure that we are not introducing a breaking change to users.

Comment (codeboten) [reply processor/groupbytraceprocessor/documentation.md:9]: the prometheus exporter configuration prepends that prefix automatically

**Review by jpkrohling — APPROVED** [review]
I'm approving this under the assumption that the metrics before and after this change are the same from the user's perspective.

**Comment by codeboten** [comment]
Metrics before the change:

```
# HELP otelcol_processor_groupbytrace_conf_num_traces Maximum number of traces to hold in the internal storage
# TYPE otelcol_processor_groupbytrace_conf_num_traces gauge
otelcol_processor_groupbytrace_conf_num_traces{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 1e+06
# HELP otelcol_processor_groupbytrace_event_latency How long the queue events are taking to be processed
# TYPE otelcol_processor_groupbytrace_event_latency histogram
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceExpired",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceReceived",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceReleased",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceRemoved",service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
# HELP otelcol_processor_groupbytrace_incomplete_releases Releases that are suspected to have been incomplete
# TYPE otelcol_processor_groupbytrace_incomplete_releases counter
otelcol_processor_groupbytrace_incomplete_releases{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_num_events_in_queue Number of events currently in the queue
# TYPE otelcol_processor_groupbytrace_num_events_in_queue gauge
otelcol_processor_groupbytrace_num_events_in_queue{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_num_traces_in_memory Number of traces currently in the in-memory storage
# TYPE otelcol_processor_groupbytrace_num_traces_in_memory gauge
otelcol_processor_groupbytrace_num_traces_in_memory{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_spans_released Spans released to the next consumer
# TYPE otelcol_processor_groupbytrace_spans_released counter
otelcol_processor_groupbytrace_spans_released{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 2
# HELP otelcol_processor_groupbytrace_traces_evicted Traces evicted from the internal buffer
# TYPE otelcol_processor_groupbytrace_traces_evicted counter
otelcol_processor_groupbytrace_traces_evicted{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_traces_released Traces released to the next consumer
# TYPE otelcol_processor_groupbytrace_traces_released counter
otelcol_processor_groupbytrace_traces_released{service_instance_id="0f498566-0172-4a71-8750-a863fb42156b",service_name="otelcontribcol",service_version="0.104.0-dev"} 1

```

After the change:

```
# HELP otelcol_processor_groupbytrace_conf_num_traces Maximum number of traces to hold in the internal storage
# TYPE otelcol_processor_groupbytrace_conf_num_traces gauge
otelcol_processor_groupbytrace_conf_num_traces{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 1e+06
# HELP otelcol_processor_groupbytrace_event_latency How long the queue events are taking to be processed
# TYPE otelcol_processor_groupbytrace_event_latency histogram
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="2000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceExpired",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="2000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceReceived",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="2000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceReleased",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="10"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="20"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="50"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="100"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="200"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="500"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="1000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="2000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="5000"} 1
otelcol_processor_groupbytrace_event_latency_bucket{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev",le="+Inf"} 1
otelcol_processor_groupbytrace_event_latency_sum{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
otelcol_processor_groupbytrace_event_latency_count{event="onTraceRemoved",service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
# HELP otelcol_processor_groupbytrace_incomplete_releases Releases that are suspected to have been incomplete
# TYPE otelcol_processor_groupbytrace_incomplete_releases counter
otelcol_processor_groupbytrace_incomplete_releases{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_num_events_in_queue Number of events currently in the queue
# TYPE otelcol_processor_groupbytrace_num_events_in_queue gauge
otelcol_processor_groupbytrace_num_events_in_queue{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_num_traces_in_memory Number of traces currently in the in-memory storage
# TYPE otelcol_processor_groupbytrace_num_traces_in_memory gauge
otelcol_processor_groupbytrace_num_traces_in_memory{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_spans_released Spans released to the next consumer
# TYPE otelcol_processor_groupbytrace_spans_released counter
otelcol_processor_groupbytrace_spans_released{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 2
# HELP otelcol_processor_groupbytrace_traces_evicted Traces evicted from the internal buffer
# TYPE otelcol_processor_groupbytrace_traces_evicted counter
otelcol_processor_groupbytrace_traces_evicted{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 0
# HELP otelcol_processor_groupbytrace_traces_released Traces released to the next consumer
# TYPE otelcol_processor_groupbytrace_traces_released counter
otelcol_processor_groupbytrace_traces_released{service_instance_id="c1fc6a3b-e1ec-4fcf-8f48-c4310d1c17ad",service_name="otelcontribcol",service_version="0.104.0-dev"} 1
```

**Comment by codeboten** [comment]
Brought back the metric test that was looking at the gauge values using otel instead of opencensus. This should be ready to go now

**Review by codeboten — APPROVED** [review]

**Pull request merged**